src/features/hotspots/setup/HotspotTxnsSubmitScreen.tsx

```diff
@@ -33,7 +37,20 @@ const HotspotTxnsSubmitScreen = () => {
       if (!gatewayTxn) {
         return
       }
-      await submitTxn(gatewayTxn)
+      const pendingTxn = await submitTxn(gatewayTxn)
+
+      // Segment track for add gateway
+      track(HotspotEvents.ADD_GATEWAY_SUBMITTED, {
+        pending_transaction: {
```

Comment:
Is it possible to get the variant that is being onboarded?

Reply:
I am adding this feature.

Reply:
yes, it's possible and now **Maker** and **variant** are added all the track calls(add-gateway, assert-location, transfer-device).